tests/make/docs_generate_success.bats

```diff
@@ -4,20 +4,27 @@ TEST_DIR="$(pwd)/docs_generate_success"
 
 # generate a test terraform project with a nested "module"
 function setup() {
+rm -rf "$TEST_DIR"
```

Comment:
somewhat curious on these changes... is `teardown()` not doing its job? `cat >` should also make `touch` unnecessary, right?

Reply:
re`touch`: I was running into some weird errors while testing things and just threw these in there without digging too deep. 

The `rm` line was from when I was going back and forth between commenting out the `teardown()` function while getting the test(s) to pass. it was useful enough and low impact enough that I just left it

Reply:
sounds like `bats` could use a debug mode, where it steps through its functions one-by-one so you can inspect the state and then continue...